packages/components/src/custom-select-control/test/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+/**
+ * WordPress dependencies
+ */
+import { CustomSelectControl } from '@wordpress/components';
+
+describe( 'CustomSelectControl', () => {
+	const onKeyDown = jest.fn();
```

Comment:
It's probably better if the `onKeyDown` spy gets instantiated _inside_ the test, to avoid potential bugs in the future when adding more tests and forgetting to reset the spy in between tests. i.e.

```js

describe( 'CustomSelectControl', () => {
  it( 'Captures the keypress event and does not let it propagate', () => {
    const onKeyDown = jest.fn();
    // ...
```

Reply:
Thanks, good point!